x/rewards/genesis.go

```diff
@@ -19,6 +19,10 @@ func InitGenesis(ctx sdk.Context, k keeper.Keeper, genState types.GenesisState)
 	if len(genState.RefillRewardsTS.TimeEntries) == 0 {
 		k.RefillRewardsPools(ctx, nil, nil)
 	}
+
+	for _, sub := range genState.IprpcSubscriptions {
```

Comment:
what about the cost?
use k.SetIprpcData

Reply:
done. see https://github.com/lavanet/lava/commit/dc29e1a1e0181b1110dac3e10733bc10915a321e